data-raw/standard_format_markdowns/cordoleani_data.Rmd

```diff
@@ -0,0 +1,198 @@
+
+-   mapped `Channel` and `River Channel` to `perennial instream`
+
+-   mapped `Wetland` to `floodplain`
+
+-   Changed units for `prey_density` from count/m\^3 to count/L
+
+```{r}
+cordoleani_zoop <- cordo_prey_raw %>% 
+  rename(prey_density = value, # units are organisms/m^3 per Sutter 2020 report 
+         sample_id = report_id,
+         date = sample_date,
+         location = region,
+         species = taxa) %>%
+  mutate(date = mdy(date),
+         prey_density = prey_density/1000, # 1 m^3 = 1000 Liters 
+         author = 'Cordoleani',
+         habitat_type = ifelse(habitat_type == "Agriculture", 'agricultural canal',
+                               ifelse(habitat_type == 'Channel', 'perennial instream',
+                                      ifelse(habitat_type == "River Channel", 'perennial instream',
+                                             ifelse(habitat_type == 'Wetland', 'floodplain', 'agricultural canal' )))),
+         watershed = case_when(location == "Butte Sink" ~ "Butte Creek", 
+                            location == "Sacramento River" ~ "Sacramento River", 
+                            location == "Feather River" ~ "Feather River", 
+                            location %in% c("Lower Bypass", "Upper Bypass") ~ 
+                              "Sutter Bypass"), 
+         site = paste0(location, "-", sample_id),
+         size_class = "meso",
+         gear_type = "net throw",
+         mesh_size = 150,
+         species = tolower(species)) %>% 
+         select(-location, -sample_id, -year) 
+
+```
+
+##### Locations Standard Format
+
+**variables removed**
+
+**notes:**
+
+```{r}
+cordoleani_locations_raw <- read_csv('../cordoleani/Cage_Locations.csv')
+
+cordoleani_locations <- cordoleani_locations_raw %>%
+  rename(location = Region, 
+         sample_id = Report_ID, 
+         habitat_type = Type, 
+         lon = long) %>%
+  select(-Name,  -Report_type) %>%
+  mutate(habitat_type = ifelse(habitat_type == "Agriculture", 'agricultural canal',
+                               ifelse(habitat_type == 'Channel', 'perennial instream',
+                                      ifelse(habitat_type == "River Channel", 'perennial instream',
+                                             ifelse(habitat_type == 'Wetland', 'floodplain', 'agricultural canal' ))))) %>%
+  mutate(site = paste0(location, "-", sample_id)) %>%
+  mutate(site_for_fish = paste0(location, "-", Site)) %>%
+  select(-location, -sample_id, -habitat_type) %>%
+  glimpse()
+
+```
+
+##### Combine prey data with locations
+
+This produces the final prey density dataset.
+
+```{r}
+
+cordoleani_zoop_final <- cordoleani_zoop |> 
+  left_join(cordoleani_locations) |> 
+  select(-site_for_fish, -Site)
+
+kable(head(cordoleani_zoop_final, 5))
+
+```
+
+#### QC
+
+**Notes:**
+
+```{r}
+summary(cordoleani_zoop_final)
+```
+
+#### Data exploration
+
+```{r}
+ggplot(cordoleani_zoop_final, aes(x = as.factor(month(date)), y = prey_density)) + 
```

Comment:
I think a box plot would be more helpful here or change the alpha on the dots so that you can see how many there are.

Also update to them minimal

Or both a box plot and this dot plot so we can see distribution and number of dots